components/net/http_loader.rs

```diff
@@ -1912,6 +1927,7 @@ async fn http_network_fetch(
 
     // Step 1: Let request be fetchParams’s request.
     let request = &mut fetch_params.request;
+    let context = Arc::new(context.clone());
```

Comment:
This shouldn't be necessary; what's the error that it's solving?

Reply:
Mmmm, I'm guessing the compiler doesn't like trying to use a `&FetchContext` from a closure that can be run on another thread. I think we need to add all the values you need from the FetchContext as arguments to send_response_values_to_devtools instead.